package.json

```diff
@@ -20,6 +20,7 @@
     "bootstrap": "./bin/bootstrap.sh",
     "build": "lerna run --parallel build",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "coverage-lcov": "nyc report --reporter=lcov",
```

Comment:
Can we add coveralls secrets to push the results to coveralls? Or later PR ? 

Also OOI why the different reporter?

Reply:
The Coveralls action looks for an LCOV file, then posts it using the `GITHUB_TOKEN` secret. This action writes the file.

Reply:
Ah - and then the `Coveralls` step will post that file..gotcha 👍

Reply:
I can remove the coverage script from Travis too. Right now it's posting twice. https://coveralls.io/repos/250962/builds

Reply:
At least, I think it's posting twice. I might need to revisit this if the coverage stops updating.